cradle-admin-tool-http/src/test/java/com/exactpro/th2/cradle/adm/http/PagesHttpTest.java

```diff
@@ -124,6 +124,140 @@ public void addPageWithParamsTest() throws Exception {
         assertEquals(comment, page.getComment());
     }
 
+    @Test
+    public void updatePageCommentTest() throws Exception {
+        String bookName = "updatePageCommentTest";
+        String pageName = "page2";
+        String pageComment = "update-page-comment-test";
+        Instant time = Instant.now().minus(2, ChronoUnit.MINUTES);
+        addBook(new BookToAdd(bookName, Instant.now()));
+
+        BookId bookId = new BookId(bookName);
+        PageId pageId = new PageId(bookId, time, pageName);
+        BookInfo bookInfo = this.storage.getBook(bookId);
+
+        HttpTester.Response response = this.executeGet(String.format("/new-page?book-id=%s&page-name=%s&page-start=%s",
+                bookName, pageName, time));
+        assertEquals(200, response.getStatus());
```

Comment:
suggestion: to understand what exactly has gone wrong we probably should add a message to assertions. Error 404 is not equal 200 does not say much

Reply:
Looks like http server response only 200 code. I will look at this in a separate PR